src/packerlicious/packer.py

```diff
@@ -0,0 +1,73 @@
+import subprocess
+import shlex
+import os
+import tempfile
+from packerlicious.template import Template
+
+class Packer(object):
+    """
+    Packer Binary Wrapper
+    """
+
+    packer_path = ""
+    template = None
+    _vars = {}
+
+    def __init__(self, template=None, _vars={}, packer_path=""):
+        if len(packer_path) > 0 and packer_path[-1] != '/':
+            packer_path += "/"
+        self.packer_path = packer_path
+        self.template = template
```

Comment:
Okay :+1: 

So each method will require `template` as parameter.
Also Should the `packer` binary path be the parameter for each method?
I did it because, I did not make assumption whether user has updated path variable w.r.t. packer.

Reply:
> So each method will require template as parameter.

correct

> Also Should the packer binary path be the parameter for each method?

I understand. I think it is fine the way it is as a class variable. A future enhancement could be to look at `os.env` variable or something if it is useful

For now I think this is a fair expectation that the user know where packer is on their system and provide the location if necessary.

The `check_available` method should catch if packer isn't found in the location

Reply:
Okay

> So each method will require template as parameter.

I think we won't require `Packer ` class. All methods will be part of `packer`.

But in this case, `packer_path` should be a parameter for each method, since we don't have class.

What do you think?
Correct me if I'm wrong.

Reply:
try it out, can always change it back. so long as you aren't making `packer_path` required for the user to provide i think it should be fine.